tests-policy

```diff
@@ -1,340 +0,0 @@
-#!/usr/bin/env python3
```

Comment:
We still use this in rhel-7.9 branch.

Reply:
Ah right, thanks for spotting! That commit doesn't backport at all, and I'm not in the mood of re-doing it, so I just dropped this commit.